magicparse/schema.py

```diff
@@ -72,30 +72,31 @@ def stream_parse(
             row_number += 1
 
         for row in reader:
-            errors = []
             row_number += 1
-            item = {}
-            for field in self.fields:
-                try:
-                    value = field.read_value(row)
-                except Exception as exc:
-                    errors.append({"row-number": row_number, **field.error(exc)})
-                    continue
+            if any(row):
```

Comment:
Est-ce que tu pourrais inverser la condition pour éviter d'augmenter l'indention stp :) ?

Reply:
Tu veux dire avec un "continue" ?

Reply:
Si tu veux éviter de mettre un if dans ta boucle, tu peux même le sortir en itérant sur un nouvel itérateur:

```python
rows = (row for row in reader if any(row))

for row in rows:
```